Lib/test/test_importlib/extension/test_loader.py

```diff
@@ -275,13 +275,18 @@ def test_bad_traverse(self):
             (Multiphase initialization modules only)
         '''
         script = """if True:
-                from test import support
-                import importlib.util as util
-                spec = util.find_spec('_testmultiphase')
-                spec.name = '_testmultiphase_with_bad_traverse'
-
-                with support.SuppressCrashReport():
-                    m = spec.loader.create_module(spec)"""
+                #  This try block will prevent exceptions from ending the
+                #  process with non-zero status
```

Comment:
At the first read, I failed to understand the purpose of the "except: pass" even with the comment. Moreover, I also missed the comment, so I started to read from the end :-D Maybe move the comment at the bottom, just before "except:"?

I suggest to explain that the test expects a crash as a non-zero exit code. Python exceptions are reported as exit code 0 (success), so the test fails.

nitpick: "#  ..." <= only one comment after the dash.

Reply:
Not really. Mishandling of m_traverse leads to segfault occasionally.
This test is trying to reproduce this specific situation and cause a C-level crash. Problem is, Python exceptions also end the process with non-zero status.

Would this comment be more clear?
"Prevent Python-level exceptions from ending the process with non-zero status (We are testing for a crash in C-code)"

Reply:
Yes, your proposed comment looks better to me.